.github/dependabot.yml

```diff
@@ -12,11 +12,13 @@ updates:
   labels:
     - "area/ci"
     - "ok-to-test"
-# Go
+
+# Main Go module
 - package-ecosystem: "gomod"
   directory: "/"
   schedule:
     interval: "weekly"
+    day: "monday"
```

Comment:
Not fridays? 🙃

Reply:
We can collect during the week and then merge on Friday :)